metadefender-plugin-server/src/main/scala/com/opswat/teamcity/ArtifactScanner.scala

```diff
@@ -146,6 +155,32 @@ class ArtifactScanner(config: MConfigManager) extends BuildServerAdapter {
                 post.setHeader("apikey", config.mAPIKey.mkString)
               }
 
+              val buildParams = runningBuild.getParametersProvider
+              val sandboxEnabled = (buildParams.get("system.metadefender_scan_artifact_with_sandbox") == "1" ||

```

Comment:
This doesn't allow to turn off sandbox scan for a given build when the global sandbox scan is enabled

Reply:
Fixed.